src/main/java/sirius/biz/jobs/batch/file/FileImportJob.java

```diff
@@ -199,15 +203,72 @@ protected void executeForArchive(String filename, FileHandle fileHandle) throws
         process.log(ProcessLog.info().withNLSKey("FileImportJob.importingZipFile"));
 
         AtomicInteger filesImported = new AtomicInteger();
+        if (entriesToExtract.isEmpty()) {
```

Comment:
Why not simply calling LinkedHashMap<String, Boolean> entriesToExtract = getEntriesToExtract()? Do we need the additional complexity of field+add+define?

Reply:
It gives the user some boiler plate code, otherwise one must manually create and return this map.